src/components/FilterPane/FilterPane.jsx

```diff
@@ -70,38 +70,37 @@ const FilterPane = ({ mermaidUserData }) => {
   const location = useLocation()
   const { isAuthenticated } = useAuth0()
   const {
-    projectData,
-    setCountries,
-    setOrganizations,
-    displayedProjects,
-    selectedCountries,
-    setSelectedCountries,
-    selectedOrganizations,
-    setSelectedOrganizations,
-    sampleDateAfter,
-    sampleDateBefore,
-    methodDataSharingFilters,
-    projectNameFilter,
     checkedProjects,
-    setCheckedProjects,
-    showYourData,
-    handleSelectedCountriesChange,
-    handleSelectedOrganizationsChange,
+    countriesSelectOnOpen,
+    displayedCountries,
+    displayedOrganizations,
+    displayedProjects,
     formattedDate,
+    getActiveProjectCount,
     handleChangeSampleDateAfter,
     handleChangeSampleDateBefore,
     handleMethodDataSharingFilter,
     handleProjectNameFilter,
+    handleSelectedCountriesChange,
+    handleSelectedOrganizationsChange,
     handleYourDataFilter,
-    userIsMemberOfProject,
-    displayedCountries,
-    setDisplayedCountries,
+    methodDataSharingFilters,
+    organizationsSelectOnOpen,
+    projectData,
+    projectNameFilter,
     remainingDisplayedCountries,
-    displayedOrganizations,
+    sampleDateAfter,
+    sampleDateBefore,
+    selectedCountries,
+    selectedOrganizations,
+    setCheckedProjects,
+    setCountries,
+    setDisplayedCountries,
     setDisplayedOrganizations,
-    countriesSelectOnOpen,
-    organizationsSelectOnOpen,
-    getActiveProjectCount,
+    setSelectedCountries,
+    setSelectedOrganizations,
+    showYourData,
+    userIsMemberOfProject,
   } = useContext(FilterProjectsContext)
```

Comment:
Any of the diffs related to a `useContext(FilterProjectsContext)` are just alphabetizing.